README.md

```diff
@@ -641,12 +641,12 @@ follow_joint_trajectory:
   type: FollowJointTrajectory
   default: true
   joints:
-   - group_1/joint_1
-   - group_1/joint_2
-   - group_1/joint_3
-   - group_1/joint_4
-   - group_1/joint_5
-   - group_1/joint_6
+   - joint_1
+   - joint_2
+   - joint_3
+   - joint_4
+   - joint_5
+   - joint_6
```

Comment:
should we perhaps include an example for multi-group setups as well?

Once/if you know, it's trivial, but that's always the case with documentation.

Reply:
I have actually not yet set up MoveIt on a multi-group setup, so I am not totally sure what this should look like. If you would like to provide an example, feel free. Either way is okay with me.

Reply:
I've made a note in https://github.com/Yaskawa-Global/motoros2/issues/74#issuecomment-2554085013